modules/admin_manual/pages/installation/index.adoc

```diff
@@ -1,5 +1,20 @@
 = Installation
 :page-aliases: go/admin-install.adoc
 
-In this section, you will find all the information you need for installing ownCloud. 
+You can install ownCloud in multiple ways, here are our trusted guides:
 
+== xref:installation/manual_installation.adoc[Manual Installation]
+
+This is a thorough guide to installing ownCloud, containing all the information needed for the prerequisites, the dependencies, the actual installation and the configuration afterwards.
+
+== xref:installation/docker/index.adoc[Installing With Docker]
+
+This guide will show you how to install ownCloud with Docker Compose using our YAML file.
+
+== xref:installation/ubuntu_18_04.adoc[Example Installation on Ubuntu]
```

Comment:
Could we change 18.04 to 20.04? I think the commands are the same with exception of the php-smbclient.

Reply:
yes, but then we also need to add a note about the Ondrej repo.